### PR TITLE
Fixing CVE-2018-18264

### PR DESCRIPTION
**Pull request by hyperbolic2346**
also fixing 1.10 builds, which is cool since I broke it :)

### CODE REVIEW

**Comment by Cynerva** [comment]
> also fixing 1.10 builds

We build cdk-addons 1.10 from the release-1.10 branch. I don't think that fix belongs in master.

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 KUBE_ARCH=amd64
-KUBE_DASHBOARD_VERSION=v1.8.3
+# need to jump to master to get 1.10.1 for CVE-2018-18264
+#KUBE_DASHBOARD_VERSION=v1.8.3
+KUBE_DASHBOARD_VERSION=master
```
Comment (Cynerva) [inline Makefile]: What's wrong with `KUBE_DASHBOARD_VERSION=v1.10.1`? I see a tag for it

**Comment by hyperbolic2346** [comment]
> > also fixing 1.10 builds
> 
> We build cdk-addons 1.10 from the release-1.10 branch. I don't think that fix belongs in master.

I blame the long vacation. I'll pull that out.

File: Makefile (hunk 1 above)
Comment (hyperbolic2346) [reply Makefile]: missed that, I just checked branches.

**Comment by Cynerva** [comment]
LGTM. Can you do the same for the release-1.11 and release-1.10 branches as well?

**Pull request merged**